### ISSUE TITLE
Can we add links to each ice-cream protocol/ flavor on the main README page?

### ISSUE THREAD

**Issue by davidbrown2324** [shuqin19/ice-cream#7]
@shuqin19 @SBEnki This is looking so good. Does this mean you are both happy with pull requests now?

**Comment by SBEnki** [comment]
yes, I think I've got a handle of forking from someone else's masters, updating my own file, and then requesting the master be updated. Then updating from the master again.

Yes, will add. See if I can get a better handle of markdown.
